.github/workflows/build.yml

```
@@ -0,0 +1,22 @@
+name: build
+on: [push, pull_request]
+jobs:
+  go:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.13.1'
+      - run: go test -v ./go/...
+
+  javascript:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15.x
```

Can we use a maintained version here?

```suggestion
          node-version: 16
```

or 

```suggestion
          node-version: lts/*
```

---

Thanks! Fixed. Let's use 'lts/*'
I referred to https://github.com/actions/setup-node